app/controllers/custom_org_links_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CustomOrgLinksController < ApplicationController
+  before_action :set_custom_org_link, only: %i[edit update destroy]
+  after_action :verify_authorized
+
+  def new
+    authorize CustomOrgLink
+    @custom_org_link = CustomOrgLink.new
+  end
+
+  def create
+    authorize CustomOrgLink
+
+    if current_organization.custom_org_links.count >= CustomOrgLink::MAX_RECORDS_PER_ORG
```

Comment:
Should this only check that "active" links are below the threshold? If so, we probably want to make the check in `update` as well.

Reply:
The way I read the technical requirements in the issue, it seemed that we wanted to cap the total number of links in the DB per org. However, there's certainly room for ambiguity. I'm happy to make the change if we want to allow more inactive links to exist. I still think some cap would be a good idea though, or we could have an unbounded number of DB records created.

Reply:
We talked to the stakeholders today and they say no cap needed

Reply:
@compwron, do we still want to limit the number of active links, or is it just free reign on whatever they want?